### PR TITLE
Fix conversion tool fsa.py for Python 3.6

### PR DESCRIPTION
**Pull request by svecjan**
Hi, I needed to convert the model to XML and it didn't work for Python 3.6, so I made a fix.
The source code mixed the spaces and tabulators, so I left the spaces there. Is ok?

### CODE REVIEW

File: xmlwriter.py [hunk 1]
```diff
@@ -17,126 +17,131 @@
...
-	self.path = []
-	self.encoding = encoding
-	encoder, decoder, streamReader, streamWriter = codecs.lookup(encoding)
-	self.file = streamWriter(file)
-	self.margin = 78
+    def __init__(self, file, encoding='UTF-8'):
+        self.path = []
+        self.encoding = encoding
+        encoder, decoder, streamReader, streamWriter = codecs.lookup(encoding)
+        self.file = sys.stdout
```
Comment (jtrmal) [inline xmlwriter.py]: is this supposed to be sys.stdout or file?

File: xmlwriter.py [hunk 2]
```diff
@@ -17,126 +17,131 @@
...
-    def __init__(self, file, encoding='ISO-8859-1'):
-	self.path = []
-	self.encoding = encoding
-	encoder, decoder, streamReader, streamWriter = codecs.lookup(encoding)
-	self.file = streamWriter(file)
-	self.margin = 78
+    def __init__(self, file, encoding='UTF-8'):
+        self.path = []
+        self.encoding = encoding
+        encoder, decoder, streamReader, streamWriter = codecs.lookup(encoding)
```
Comment (jtrmal) [inline xmlwriter.py]: this is also weird... the lifetimes(scopes) of these four variables  are limited to __init__, no?

File: xmlwriter.py [hunk 3]
```diff
@@ -17,126 +17,131 @@
...
+            out = [self.fillParagraph(line) for line in w.split('\n')]
+            out = reduce(operator.add, out)
+        elif format == self.formatFill:
+            out = self.fillParagraph(w)
+        self.write("\n".join(out) + '\n')
 
     def formatted_cdata(self, s):
-	for w in string.split(s, u'\\n'):
-	    self.cdata(w, self.formatFill)
+        for w in " ".split(s, '\\n'):
```
Comment (jtrmal) [inline xmlwriter.py]: also suspicious -- I would assume s.split('\\n')?
Could you check your code once more to makes sure these mistakes are gone?

Comment (jtrmal) [reply xmlwriter.py]: I doubt I would be able to see all of it

**Comment by jtrmal** [comment]
Thanks! Spaces are definitely prefered.

File: xmlwriter.py (hunk 3 above)
Comment (jtrmal) [reply xmlwriter.py]: and by '\n' I meant r'\n' or '\\\\n' -- github eats the slashes in comments

File: xmlwriter.py (hunk 2 above)
Comment (honzas83) [reply xmlwriter.py]: The line should be definitely removed, it contains dead code

**Comment by jtrmal** [comment]
@svecjan are you planning to work on the code? Let me know.

**Comment by jtrmal** [comment]
Testing something else -- closing and reopening

**Comment by jtrmal** [comment]
closing -- resolved by  #55

**Pull request closed without merging**